recipes/methpype/meta.yaml

```
@@ -0,0 +1,39 @@
+package:
+  name: methpype
+  version: 1.0.1
+
+source:
+  url: https://files.pythonhosted.org/packages/00/88/eab68b60ad237ba0f9b69f862b6b92553a9646b4e5620a06e58803fa645d/methpype-1.0.1.tar.gz
```

Could you please change it to:
```suggestion
  url: https://files.pythonhosted.org/packages/00/88/eab68b60ad237ba0f9b69f862b6b92553a9646b4e5620a06e58803fa645d/methpype-{{ version }}.tar.gz
```